### ISSUE TITLE
Redesign model

### ISSUE THREAD

**Issue by zebulonmcknight** [zebulonmcknight/cs325_final#2]
- Choose between scratch and sklearn log_reg 
- Add model analysis tools (loss functions and confusion matrices 
- Consider options other than log_reg maybe 
- Configure parameters and arguments
